backup/src/main/java/com/instaclustr/backup/task/TaskFactory.java

```diff
@@ -0,0 +1,5 @@
+package com.instaclustr.backup.task;
+
+public interface TaskFactory {
+    BackupTask backupTask();
```

Comment:
Is this factory used? Perhaps it should also create `RestoreTask`s?

Reply:
Deleted